packages/commerce-sdk-react/src/hooks/ShopperLogin/helper.test.tsx

```diff
@@ -11,11 +11,16 @@ import {renderWithProviders} from '../../test-utils'
 
 const mockLoginGuestUser = jest.fn().mockResolvedValue('mockLoginGuestUser')
 
-jest.mock('../useAuth', () => {
-    return jest.fn(() => ({
-        ready: () => Promise.resolve({access_token: '123'}),
-        loginGuestUser: mockLoginGuestUser
-    }))
+jest.mock('../../auth', () => {
```

Comment:
> Does ../useAuth (the bit you're replacing) exist?

Yup, the useAuth hook still exist. We don't need to mock it because it will eventually use the underlying Auth instance, which is going to be mocked this time around.

The problem previously was that when we called `renderWithProviders`, our commerce Provider would call `auth.ready()` directly (to fetch the access token for you) and not via the useAuth hook. So we needed to mock the underlying Auth instance instead.

Reply:
> Any idea why this didn't fail locally?

I'm not sure... Previously, the test was passing locally, but you can see there's some error logged. The strange things was locally the error didn't seem to matter, and yet on CI it's causing the test to fail.

![NeoVim Terminal 2022-11-09 at 16 00 21](https://user-images.githubusercontent.com/847300/200968420-84cf32f6-fe0e-416a-9197-d50184f4fecd.png)